query-engine/connector-test-kit/src/test/scala/writes/topLevelMutations/UpdateMutationSpec.scala

```diff
@@ -161,8 +161,8 @@ class UpdateMutationSpec extends FlatSpec with Matchers with ApiSpecBase {
          |  }
          |}""".stripMargin,
       project,
-      errorCode = 3039,
-      errorContains = "No Node for the model Todo with value NOT A VALID ALIAS for alias found"
+      errorCode = 0, // 3039,
+      errorContains = """InterpretationError(\"Error for binding \\'0\\': RecordNotFound(\\\"Record to update not found.\\\")\""""
```

Comment:
Probably not something users can/should handle but we could return a better error message here probably.

Reply:
yes